docs/source/datasets/index.rst

```diff
@@ -499,8 +499,28 @@ Tax form: 1040
 As with data collected from W-2 and 1099 forms, pseudopeople enables the simulation of administrative records from 1040 forms, which are
 also reported to the IRS on an annual basis. To find out more about the 1040 tax form, visit the `IRS information page <https://www.irs.gov/instructions/i1040gi>`_.
 
-Generate 1040 data with :func:`pseudopeople.generate_taxes_1040`.
+A single row in a pseudopeople-generated 1040 dataset may contain information about several
+simulants: the primary filer, the primary filer's joint filer (spouse) if they are married filing
+jointly, and up to four claimed dependents.
```

Comment:
Somewhere, perhaps after this sentence, we should probably say that `simulant_id` and `household_id`, and actually all columns before `spouse_first_name`, apply to the *filer*, not the joint filer or the dependents. We could perhaps comment on this in the Notes column of the table if that would be clearer (but we might get merge conflicts with Abie's PR if we do that here).

(Also, I just saw a note I made from the doc session that maybe we should add simulant IDs for spouse and dependents, but that should wait until after yellow release -- just mentioning it here in case it's relevant to how we describe things.)

Reply:
I've added a note about this above the table.